src/main/java/jewellery/inventory/service/OrganizationService.java

```diff
@@ -67,6 +80,37 @@ private List<Organization> getAll() {
     return organizationRepository.findAll();
   }
 
+  private void validateOrganizationProductsAndResources(Organization organization) {
+    if (!organization.getProductsOwned().isEmpty()) {
+      throw new OrganizationProductsException(organization.getId());
+    }
+    if (!organization.getResourceInOrganization().isEmpty()) {
+      throw new OrganizationResourcesException(organization.getId());
+    }
+  }
+
+  public void validateCurrentUserPermission(
```

Comment:
Why this is public? Do we need it outside of this class ?

Reply:
yes we use them in UserInOrganizationService